FXIOS-1490 and FXIOS-1491 - Native Library Tray

nbhasin2 opened the pull request:
This is mostly a visual refactoring of library tray to use native view instead of custom bottom sheet.

<img width="419" alt="Screen Shot 2021-04-26 at 5 27 27 PM" src="https://user-images.githubusercontent.com/8919439/116153019-aeaf9a00-a6b4-11eb-8f11-379ee0c204a6.png">

dnarcese commented:
Can you pull main in again to test this with the new menu which splits out "Library"?

nbhasin2 commented:
@dnarcese I pulled main into this branch and your new menu opens library properly 

https://user-images.githubusercontent.com/8919439/116296548-5c2fb580-a768-11eb-9c73-07110ee0f91b.mov

nbhasin2 commented:
These are the tests that are failing on BR but are passing locally. Somehow `Example.com` wasn't reachable but then I changed my DNS and it worked so not sure if that has something to do with it. Added a 1 sec sleep as `waitForExistence` didn't work for `testAllOptionsArePresent`

Going to tray again will likely merge it in and make a ticket to fix these 🤞🏼 

<img width="671" alt="Screen Shot 2021-04-27 at 5 10 37 PM" src="https://user-images.githubusercontent.com/8919439/116313229-7de66800-a77b-11eb-84ca-0b59eb8c11fb.png">

<img width="812" alt="image" src="https://user-images.githubusercontent.com/8919439/116329169-b6486f00-a798-11eb-93cd-502a911411ee.png">
